src/app/clusters/account-login-server/account-login-server.cpp

```diff
@@ -64,16 +75,9 @@ namespace app {
 namespace Clusters {
 namespace AccountLogin {
 
-void SetDefaultDelegate(EndpointId endpoint, Delegate * delegate)
+void SetDefaultDelegate(Delegate * delegate)
```

Comment:
Why rollback multi  endpoint support?

Reply:
There is a default delegate which the main tv code assigns. This would be the handler for the cluster instances on endpoint 1.

All other cluster instances for the media clusters would be on a content app, and the ContentAppPlatform manages the mapping from endpoint to implementation, with a single method to add and remove an endpoint as a ContentApp is loaded and unloaded

Reply:
per slack discussion, added endpoint argument back